jenkinsci/neuvector-vulnerability-scanner-plugin
# [NVSHAS-8938] Add Critical CVE Severity to support CVSS v3 scores 9.0-10.0

### Summary
1. Add Critical CVE Severity to support CVSS v3 scores 9.0-10.0

### How to verify
update from the previous version, say previous version have 5 High, but in CVSS 3.0 is 4 High 1 Critical
And current threshold for high is 5.
1. if user didn't update the pipeline script => it should hold the previous result, which is fail
<img width="1326" alt="Screen Shot 2024-05-03 at 2 54 06 PM" src="https://github.com/jenkinsci/neuvector-vulnerability-scanner-plugin/assets/145627854/60775da7-79bb-4ab5-9c23-d70cb15e678f">
2. if user update the pipeline scripts and remain the high is 5, no setting for others => it should pass.
<img width="278" alt="Screen Shot 2024-05-03 at 2 55 05 PM" src="https://github.com/jenkinsci/neuvector-vulnerability-scanner-plugin/assets/145627854/9589cdbf-883e-4b1b-98ed-d33a421b17fa">
